external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/SingleTopicKafkaSpoutTest.java

```diff
@@ -54,18 +54,20 @@ public SpoutContext(KafkaSpout<String, String> spout,
     void populateTopicData(String topicName, int msgCount) {
         kafkaUnitRule.getKafkaUnit().createTopic(topicName);
 
+        kafkaUnitRule.getKafkaUnit().createProducer();
```

Comment:
It would be nicer if the producer were created/closed in setup/teardown for the Rule